acme/acme.go

```diff
@@ -448,7 +477,7 @@ func dnsOverrideDelay(delay int) error {
 			return true, nil
 		}
 	} else if delay < 0 {
-		err = fmt.Errorf("Invalid negative DelayDontCheckDNS: %d", delay)
+		err = fmt.Errorf("invalid negative DelayDontCheckDNS: %d", delay)
```

Comment:
How come some of these messages start with a capital letter, and others don't?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings